ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -45,14 +45,10 @@ define(['QUnit', 'jquery', 'manage_taxonomies', 'reactaddons',
    * */
   function  assertAddTermCommon(assert, component) {
     // check term title
-    var listOfLinks = React.addons.TestUtils.
-      scryRenderedDOMComponentsWithTag(
-        component,
-        'a'
-    );
-    assert.equal(listOfLinks.length, 2);
-    var linkHeader = React.findDOMNode(listOfLinks[0]);
-    assert.equal(linkHeader.innerHTML, vocabulary.name);
+    var node = React.findDOMNode(component);
+    var $vocabTitles = $(node).find(".vocab-title");
+    assert.equal($vocabTitles.length, 1);
+    assert.equal($vocabTitles[0].innerHTML, vocabulary.name);
```

Comment:
Because we are using jquery in this file for example methods like find(),mock ajax etc. jquery looks more organise. 
By the way this is my suggestion.